src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -785,6 +833,24 @@ final case class AgreementApiServiceImpl(
     verifyConflictingAgreements(agreement, conflictingStates)
   }
 
+  def createAndCopyDocumentsForClonedAgreement(
+    oldAgreement: AgreementManagement.Agreement,
+    newAgreement: AgreementManagement.Agreement
+  )(implicit contexts: Seq[(String, String)]): Future[Seq[AgreementManagement.Document]] = {
+    Future.traverse(oldAgreement.consumerDocuments)(doc => {
```

Comment:
very minor, just to match our conventions
```suggestion
    Future.traverse(oldAgreement.consumerDocuments){ doc => 
```